A few changes for Crazy Fast games

muracevic opened the pull request:
Tweaked Crazy Fast village settings a bit. 
- 8 villages per player
- villages are being built faster and they give resources (update) every 5 minutes now

muracevic committed:
A few changes for Crazy Fast games

muracevic committed:
more CF tweaks

muracevic committed:
Dominus time on CF

Lowered Dominus time before win to 1h

muracevic committed:
starting Capital garrision increased for CF

muracevic committed:
Income update interval back to 10min

muracevic committed:
Village percentage income for CF

Capitals now get 12% of each village's income

dan335 commented on packages/dominus-settings/settingsCapitals.js:
```diff
@@ -7,15 +7,6 @@ _gs.capitals = function(gameId, path) {
   // game specific settings
 
   if (game && game.isSpeed) {
-    settings.income = {
```
Did you mean to remove this?  This is a change to speed games, not crazyfast.

muracevic replied:
Fixed with https://github.com/dan335/dominus/pull/2/commits/849dc7c0fd99cdc7e815eeeac1c6f6851ffe473c

dan335 commented on packages/dominus-settings/settingsCapitals.js:
```diff
@@ -29,7 +20,16 @@ _gs.capitals = function(gameId, path) {
+    }    
+
     settings.incomeInterval = 1000 * 60 * 10;
+    settings.villagePercentageIncome = 0.12;
```
Need to fix all places to use _gs instead of _s for villagePercentageIncome.  

https://github.com/dan335/dominus/blob/master/packages/dominus-capitols/client/rp_info_capitol.html#L127

help_capitals.html

Search code for villagePercentageIncome to make sure you get them all.

muracevic replied:
Ugh, seems like https://github.com/dan335/dominus/pull/2/commits/361ad406c333e4ab9eaa8bc768933f57453de006#diff-1d09e0b2953b183777c26e43b4b3b42aR33 is breaking income job. Reverting back in a minute.

muracevic replied:
Reverted with https://github.com/dan335/dominus/pull/2/commits/3ba0381f06773c0d8a27529b3583c123feea53e2 . Game running for 20 minutes now, no errors.

dan335 reviewed (changes requested):
Looks good, just a couple things.

muracevic committed:
Fixes Capitals income on Speed games

muracevic committed:
Now displays Income percentage correctly

muracevic committed:
Fixing income job

Game running with no errors anymore. I guess there was no need to change this. (?)
